### ISSUE TITLE
Uncaught (in promise) Error: This is not a registered game setting

### ISSUE THREAD

**Issue by atomdmac** [Brownie79/fvtt-swade-stat-block-parser#8]
Foundry VTT Version: `0.6.5`
SWADE Statblock Importer Version: `1.2.5`

I get the following error in the browser console when attempting to import a stat block:

```
foundry.js:3030 Uncaught (in promise) Error: This is not a registered game setting
```

The accompanying call stack is:

  | get | @ | foundry.js:3030
-- | -- | -- | --
  | searchCompendiumsForItem | @ | importActorItems.js:203
  | importActorSkillsList | @ | importActorItems.js:173
  | importActorItems | @ | importActorItems.js:5
  | async function (async) |   |  
  | buildActor | @ | buildActor.js:23
  | callback | @ | statblock-importer.js:43
  | submit | @ | foundry.js:13537
  | _onClickButton | @ | foundry.js:13498
  | dispatch | @ | jquery.min.js:2
  | v.handle | @ | jquery.min.js:2

Examples of stat blocks that I have tried are the "Prairie Tick" from Deadlands and the Dire Wolf from the SWADE bestiary.

The console also shows that the importer was able to (at least in part) parse the stat block properly:
```
Statblock Importer | {"name":"Direwolf","type":"npc","description":"","attributes":{"agility":{"die":{"sides":8,"modifier":0}},"smarts":{"die":{"sides":4,"modifier":0}},"spirit":{"die":{"sides":6,"modifier":0}},"strength":{"die":{"sides":8,"modifier":0}},"vigor":{"die":{"sides":8,"modifier":0}}},"skills":{"Athletics":{"sides":8,"modifier":""},"Fighting":{"sides":8,"modifier":""},"Intimidation":{"sides":8,"modifier":""},"Notice":{"sides":6,"modifier":""},"Stealth":{"sides":8,"modifier":""}},"toughness":{"value":6,"armor":0,"modifier":0},"parry":{"value":6,"modifier":0},"pace":10,"hindrances":{},"edges":["Alertness"],"gear":{},"powers":{},"specials":{}}
```

**Comment by spacemandev-git** [comment]
The importer currently only works with Foundry version 0.6.2, the registered game setting bug I will fix for 0.7

**Comment by spacemandev-git** [comment]
this error should be fixed with 1.2.6 of the importer
